src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -433,10 +433,6 @@ public new LayoutItem Layout
             set
             {
                 ContentContainer.Layout = value;
-                if (ContentContainer.Layout != null)
```

Comment:
Is this change related to `TransformedByLayout` property? 
This seems to change the behavior of `ScrollableBase`.

Reply:
You are right. I fixed it with the first comment above.